terraform/itap-piem.alpha-canada.ca.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_route53_record" "itap-piem-alpha-canada-ca-A" {
+  zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+  name    = "itap-piem.alpha-canada.ca"
```

Comment:
I believe this should be `itap-piem.alpha.canada.ca`

Reply:
It has to be, nice catch I didn't even notice.